src/components/TrapFocus/tests/TrapFocus.test.tsx

```diff
@@ -10,16 +10,8 @@ import {
 import TrapFocus from '../TrapFocus';
 
 describe('<TrapFocus />', () => {
-  let requestAnimationFrameSpy: jest.SpyInstance;
-
-  beforeEach(() => {
-    requestAnimationFrameSpy = jest.spyOn(window, 'requestAnimationFrame');
-    requestAnimationFrameSpy.mockImplementation((cb) => cb());
-  });
-
   afterEach(() => {
     (document.activeElement as HTMLElement).blur();
```

Comment:
Technically we can remove this as well, how if we're expecting on the activeElement it's good practice.